GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govuksummarylistitem.config

```diff
@@ -58,7 +58,7 @@
       <Name>Value</Name>
       <Alias>itemValue</Alias>
       <Definition>0729c57a-748f-482d-abbc-fcd2976c9fc6</Definition>
```

Comment:
Does it matter that the definition hasn't changed for this and govuktask, tprcontextbar1 - 3, tprfooter and tprheader but has for the other content types above?

Reply:
Well spotted, but no it doesn't because they already had their own data types.